linchpin/shell/__init__.py

```diff
@@ -267,8 +275,10 @@ def up(ctx, targets, run_id, tx_id, inventory_format):
 @click.option('-t', '--tx-id', metavar='tx_id', type=int,
               help='Destroy resources using the transaction ID (tx-id)',
               cls=MutuallyExclusiveOption, mutually_exclusive=["run_id"])
+@click.option('--ifh', '--ignore-failed-hooks', is_flag=True)
+@click.option('--nh', '--no-hooks', is_flag=True)
```

Comment:
These should have a help kwarg with a good short description.

Reply:
See the other comment I just made above. Same, but this one is a bit more specific.